src/results/heating_cooling_load.jl

```diff
@@ -0,0 +1,84 @@
+#
+# Redistribution and use in source and binary forms, with or without modification,
+# are permitted provided that the following conditions are met:
+#
+# Redistributions of source code must retain the above copyright notice, this list
+# of conditions and the following disclaimer.
+#
+# Redistributions in binary form must reproduce the above copyright notice, this
+# list of conditions and the following disclaimer in the documentation and/or other
+# materials provided with the distribution.
+#
+# Neither the name of the copyright holder nor the names of its contributors may be
+# used to endorse or promote products derived from this software without specific
+# prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+# IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+# INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+# BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+# DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+# LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+# OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+# OF THE POSSIBILITY OF SUCH DAMAGE.
+# *********************************************************************************
+"""
+`CoolingLoad` results keys:
+- `load_series_ton` vector of site cooling load in every time step
+- `annual_calculated_tonhr` sum of the `load_series_ton`
+"""
+function add_cooling_load_results(m::JuMP.AbstractModel, p::REoptInputs, d::Dict; _n="")
+    # Adds the `ElectricLoad` results to the dictionary passed back from `run_reopt` using the solved model `m` and the `REoptInputs` for node `_n`.
+    # Note: the node number is an empty string if evaluating a single `Site`.
+
+    r = Dict{String, Any}()
+
+    load_series_kw = p.s.cooling_load.loads_kw_thermal
+    r["load_series_ton"] = load_series_kw/ KWH_THERMAL_PER_TONHOUR
+
+    r["annual_calculated_tonhr"] = round(
+        sum(r["load_series_ton"]) / p.s.settings.time_steps_per_hour, digits=2
+    )
+    
+    d["CoolingLoad"] = r
+    nothing
+end
+
+"""
+`HeatingLoad` results keys:
+- `dhw_load_series_mmbtu` vector of site domestic hot water load in every time step
+- `space_heating_load_series_mmbtu` vector of site space heating load in every time step
+- `load_series_mmbtu` vector of sum heating load in every time step
+- `annual_calculated_dhw_mmbtu` sum of the `dhw_load_series_mmbtu`
+- `annual_calculated_space_heating_mmbtu` sum of the `space_heating_load_series_mmbtu`
+- `annual_calculated_mmbtu` sum of the `load_series_mmbtu`
+"""
+function add_heating_load_results(m::JuMP.AbstractModel, p::REoptInputs, d::Dict; _n="")
```

Comment:
Let's add the *fuel* load series as well for space and dhw, as we do in the API [here](https://github.com/NREL/REopt_API/blob/5ac150780fc6759dd8a3c556a21907617325dce5/reo/nested_outputs.py#L402).

Reply:
And the annual *fuel* load energy value.

Reply:
addressed in: ea7bcad8029f07818c39d496be45284d00a747a5

I tried to match the naming conventions in the API as best I could, noting that we now have separate measures for dhw, space heating, and total.

Reply:
Great, and I just added some more verbosity by adding "thermal" to the thermal heating load names. I did this because you can't tell based on the units if it's thermal or fuel, whereas in cooling/chiller you can. I know you could know by the omission of "fuel", but I thought this was more consistent to have both "thermal" and "fuel".